README.md

```diff
@@ -37,3 +37,45 @@ All methods, with `public` visibility, are part of the public API. All other met
 ## Reporting Issues
 
 Please [create an issue](http://github.com/silverstripe/silverstripe-admin/issues) for any bugs you've found, or features you're missing.
+
+## Customising the File "Used on" table in the Files section (asset-admin)
```

Comment:
docs.silverstripe.org?  I don't believe that's the correct place to document module specific things?  If it is and I've missed something, could you please show me where it's supposed to go alongside other module specific docs?

Reply:
doc.silverstripe.org *is* the correct place for core module specific docs, otherwise we fragment the hell out of the information. Stuff that's not in core (e.g. `silverstripe/blog`) should be documented on the repo, but that's simply because we haven't had the time to pull all of this into doc.silverstripe.org. In an ideal world, doc.silverstripe.org would be your starting point for anything in supported modules, but that's hard to create a consistent IA around.

Reply:
I can't find a relevant section on docs.silverstripe.org to put information about implementing an extension for an admin feature.  Where should it go?

Reply:
Maybe as a separate article under https://docs.silverstripe.org/en/4/developer_guides/files/ ?